cmd/mqtt/main.go

```diff
@@ -35,35 +36,30 @@ const (
 	defLogLevel = "error"
 	envLogLevel = "MF_MQTT_ADAPTER_LOG_LEVEL"
 	// MQTT
-	defMQTTHost             = "0.0.0.0"
 	defMQTTPort             = "1883"
 	defMQTTTargetHost       = "0.0.0.0"
 	defMQTTTargetPort       = "1883"
-	defMQTTForwarderTimeout = "30" // in seconds
-
-	envMQTTHost             = "MF_MQTT_ADAPTER_MQTT_HOST"
+	defMQTTForwarderTimeout = "30s" // 30 seconds
 	envMQTTPort             = "MF_MQTT_ADAPTER_MQTT_PORT"
 	envMQTTTargetHost       = "MF_MQTT_ADAPTER_MQTT_TARGET_HOST"
 	envMQTTTargetPort       = "MF_MQTT_ADAPTER_MQTT_TARGET_PORT"
 	envMQTTForwarderTimeout = "MF_MQTT_ADAPTER_FORWARDER_TIMEOUT"
 	// HTTP
-	defHTTPHost       = "0.0.0.0"
 	defHTTPPort       = "8080"
 	defHTTPScheme     = "ws"
 	defHTTPTargetHost = "localhost"
 	defHTTPTargetPort = "8080"
 	defHTTPTargetPath = "/mqtt"
-	envHTTPHost       = "MF_MQTT_ADAPTER_WS_HOST"
 	envHTTPPort       = "MF_MQTT_ADAPTER_WS_PORT"
-	envHTTPScheme     = "MF_MQTT_ADAPTER_WS_SCHEMA"
+	envHTTPScheme     = "MF_MQTT_ADAPTER_WS_SCHEME"
 	envHTTPTargetHost = "MF_MQTT_ADAPTER_WS_TARGET_HOST"
 	envHTTPTargetPort = "MF_MQTT_ADAPTER_WS_TARGET_PORT"
 	envHTTPTargetPath = "MF_MQTT_ADAPTER_WS_TARGET_PATH"
 	// Things
 	defThingsAuthURL     = "localhost:8181"
-	defThingsAuthTimeout = "1" // in seconds
+	defThingsAuthTimeout = "1s" // 1 second
```

Comment:
Why is this extension `s` needed?

Reply:
Because it specifies the time unit. It's clear from the value that this is 1 second. On the opposite, having 1 is not self-documenting.

Reply:
Very interesting, I did not know about this `ParseDuration()`. Nicely solved.